mlir/include/mlir/Dialect/Linalg/IR/LinalgNamedStructuredOps.yaml

```diff
@@ -4024,6 +4024,145 @@ structured_op: !LinalgStructuredOpConfig
                 - !ScalarExpression
                   scalar_arg: K
 --- !LinalgOpConfig
+metadata: !LinalgOpMetadata
+  name: conv_3d_ncdhw_fcdhw_q
+  cpp_class_name: Conv3DNcdhwFcdhwQOp
+  doc: |-
+    Performs 3-D convolution with zero point offsets.
+
+    Numeric casting is performed on the operands to the inner multiply, promoting
```

Comment:
s/is performed on the operands to the inner multiply/
  is performed on the operands of the inner multiply ?